src/test/java/no/nav/rekrutteringsbistand/sms/rekrutteringsbistandsms/HentSmsStatuserTest.kt

```diff
@@ -45,28 +46,28 @@ class HentSmsStatuserTest {
         Thread.sleep(1500)
 
         val respons: ResponseEntity<List<SmsStatus>> = restTemplate.exchange(
-                "$baseUrl/sms/${enSmsTilOppretting.kandidatlisteId}",
-                HttpMethod.GET,
-                HttpEntity(null, null),
-                object : ParameterizedTypeReference<List<SmsStatus>>() {}
+            "$baseUrl/sms/${enSmsTilOppretting.kandidatlisteId}",
+            HttpMethod.GET,
+            HttpEntity(null, null),
+            object : ParameterizedTypeReference<List<SmsStatus>>() {}
         )
         assertThat(respons.statusCode).isEqualTo(HttpStatus.OK)
         respons.body!!.forEachIndexed { index, smsStatus ->
             assertThat(smsStatus.id).isGreaterThan(0)
             assertThat(smsStatus.fnr).isEqualTo(enSmsTilOppretting.fnr[index])
-            assertThat(smsStatus.opprettet).isEqualToIgnoringSeconds(LocalDateTime.now())
-            assertThat(smsStatus.sendt).isEqualToIgnoringSeconds(LocalDateTime.now())
+            assertThat(smsStatus.opprettet).isCloseTo(now(), within(2, SECONDS))
```

Comment:
Hva er fordelen med å bruke SECONDS framfor LocalDateTime her?

Reply:
LocalDate kan ikke brukes som parameter der. Linja asserter at smsStatus.opprettet er lik now() pluss/minus to sekunder. Dvs er omtrentlig lik. Nødvendig av to grunner: Forskjell Mac/Windows og fordi vi ikke har noen måte å vite nøyaktig hva smsStatus.opprettet er (det er ikke synlig fra testkoden).